docs/backend/configuration.md

```diff
@@ -40,3 +41,43 @@ object and viewed / modified via `/internal/config` (see
 [Accessing the admin panel](./admin.md)).
 
 ![Screenshot of site configuration admin interface](https://user-images.githubusercontent.com/47985/73627238-6276d500-467e-11ea-8724-afb703f056bc.png)
+
+## Vault
+
+The vault Ruby gem allows us to interact with
+[HashiCorp's Vault](https://www.vaultproject.io/docs/what-is-vault). In a
```

Comment:
```suggestion
[Vault](https://www.vaultproject.io/docs/what-is-vault). In a
```

Minor, but `The vault Ruby gem allows us to interact with HashiCorp's Vault` reads to me like "it's a gem that lets us interact with _their_ vault". If we just ✂️ the `HashiCorp's` here, that potential confusion will be removed.